RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# Release
+
+osd-network-verifier will follow semantic versioning
+
+## Considerations around changes
+
+These changes to these don't automatically mean a change is a breaking or significant change, but should be taken into consideration:
```

Comment:
> These changes to these don't automatically mean a change is a breaking or significant change, but should be taken into consideration

I'm not really sure what this sentence is saying. Could you clarify the wording?

Reply:
Ah, was debating between two things and ended up with nothing :D

Re-worded, but the intent was to say, these changes are things to watch out for and are probably breaking changes, but not necessarily.